presto-orc/src/test/java/com/facebook/presto/orc/BenchmarkSelectiveStreamReaders.java

```diff
@@ -35,8 +35,8 @@
 import com.google.common.base.Strings;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
-import com.google.common.primitives.Longs;
 import io.airlift.units.DataSize;
+import javafx.util.Pair;
```

Comment:
javafx is typically not used in this repo - and leads to build incompatibilities for some environments.
Maybe change this to "org.apache.commons.lang3.tuple.Pair" ? @yingsu00 @mbasmanova

Reply:
@ClarenceThreepwood Nice catch. Yes, we shouldn't use javafx.util.Pair. org.apache.commons.lang3.tuple.Pair is better.

Reply:
CC: @sujay-jain

Reply:
I've added a patch here that fixes this issue. Please take a look